expression/builtin_time_vec_test.go

```diff
@@ -27,8 +27,10 @@ import (
 var vecBuiltinTimeCases = map[string][]vecExprBenchCase{
 	ast.DateLiteral: {},
 	ast.DateDiff:    {},
-	ast.TimeDiff:    {},
-	ast.DateFormat:  {},
+	ast.TimeDiff: {
+		{retEvalType: types.ETDuration, childrenTypes: []types.EvalType{types.ETDuration, types.ETDuration}},
```

Comment:
Please add some test cases which can generate the datatime format. Such as:
```
mysql> SELECT TIMEDIFF('2000:01:01 00:00:00',
    ->                 '2000:01:01 00:00:00.000001');
        -> '-00:00:00.000001'
```

Reply:
I will try it.

Reply:
@Reminiscent 
Hi, i think `builtinDurationDurationTimeDiffSig` doesn't supports `ETDuration` arguments.